dll/win32/msgina/shutdown.c

```diff
@@ -15,6 +15,7 @@
 #define WLX_SHUTDOWN_STATE_LOGOFF       0x01
 #define WLX_SHUTDOWN_STATE_POWER_OFF    0x02
 #define WLX_SHUTDOWN_STATE_REBOOT       0x04
+#define WLX_SHUTDOWN_STATE_NT_MODE      0x08
 // 0x08
```

Comment:
You'd want to remove this comment, since you filled in the bit.

Reply:
N/A. Changed #define value to 0x100